tsconfig-strict.json

```diff
@@ -369,8 +369,6 @@
     "core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service.spec.ts",
     "core/templates/pages/exploration-player-page/exploration-player-page.constants.ts",
     "core/templates/pages/exploration-player-page/layout-directives/correctness-footer.component.ts",
-    "core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts",
-    "core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.spec.ts",
```

Comment:
As we need to get this PR merged quickly so that it can be hotfixed, I have removed these 2 files as importing question-player-state-service in exploration-footer-component resulted in multiple failures from multiple files mainly related to question-player-state-service. I will fix these and them to the list ASAP. Opened issue: https://github.com/oppia/oppia/issues/14231

Reply:
Ok!